<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Constantes_robot</class>
 <widget class="QWidget" name="Constantes_robot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constantes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/icone/imagens/oie_transparent.png</activeon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Constantes - Zona Morta</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_37">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Zona Morta_E</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rb1_ze"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Zona Morta_D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rb1_zd"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>121</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_35">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Zona Morta_E</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rb2_ze"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_36">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Zona Morta_D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rb2_zd"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Zona Morta_E</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rb3_ze"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_32">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Zona Morta_D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rb3_zd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Constantes - Controlador</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_38">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>k_lin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb1_klin">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>k_ang</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb1_kang">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>W_m치x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb1_wmax">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>V_m치x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb1_vmax">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>116</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>k_lin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb2_klin">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_27">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>k_ang</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb2_kang">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>W_m치x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb2_wmax">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>V_m치x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb2_vmax">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>116</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_29">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>k_lin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb3_klin">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>k_ang</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb3_kang">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_30">
          <item>
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>W_m치x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb3_wmax">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_28">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>V_m치x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rb3_vmax">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>horizontalSpacer_3</zorder>
     <zorder>horizontalSpacer_4</zorder>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Robo 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Robo 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>152</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Selecionar Robo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="id_rb1"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>152</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Selecionar Robo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="id_rb2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>152</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Selecionar Robo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="id_rb3">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Robo 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>horizontalSpacer_6</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
